src/Microsoft.DotNet.VersionTools.net45/Automation/GitHubApi/GitHubClient.Desktop.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net;
+
+namespace Microsoft.DotNet.VersionTools.Automation.GitHubApi
+{
+    public partial class GitHubClient : IGitHubClient, IDisposable
```

Comment:
Did you try out that `/t:UpdateDependencies` flow we talked about that might only use `BuildInfo`, not `GitHubClient`, and see it wasn't needed?

Reply:
Now that the GitHub change is live, I confirmed locally that the `raw.githubusercontent.com` endpoint still works with TLS 1.0--which means it should be fine to only have the setting in `GitHubClient`.